pkgs/by-name/wi/winbox4/package.nix

```diff
@@ -49,16 +64,28 @@ stdenvNoCC.mkDerivation (finalAttrs: {
     zlib
   ];
 
-  installPhase = ''
-    runHook preInstall
+  installPhase =
+    if stdenvNoCC.isDarwin then
```

Comment:
Similar to @drupol's comment above, I think this should use `lib.optionalString`